src/App.js

```diff
@@ -1,26 +1,22 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/no-unescaped-entities */
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import ReactDOM from 'react-dom';
+import ButtonPanel from './components/ButtonPanel';
+import Display from './components/Display';
+import './style.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default class App extends React.Component {
+  render() {
+    return (
+      <div>
+        <Display />
+        <ButtonPanel />
+      </div>
+    );
+  }
```

Comment:
Since on this milestone this should be a stateless component, you might want to make this a [functional component](https://reactjs.org/docs/components-and-props.html#function-and-class-components) and move it to the components directory:

```js
const App = () => (
  <div>
    <Display />
    <ButtonPanel />
  </div>
);
```